### PR TITLE
Add powered by badge

### PR DESCRIPTION
**Pull request by nmattia**
This adds the "Powered By" badge to the bottom of the login and
management pages.

https://user-images.githubusercontent.com/6930756/150844321-58fa726e-3c6e-40d6-afcf-959467521873.mp4

In addition:

* `<div id="navbar">` is replaced with a `<nav>`
* The `img` top-level selector had to be removed, and the attributes
  where inlined where necessary
* Some CSS classes were renamed for consistency (`navbar-link` ->
  `nav-link`)
* Some feedback was added to the navigation links, and the links were
  moved inside the app container (otherwise it looked weird with the
badge)

### CODE REVIEW

File: src/frontend/src/styles/main.css [hunk 1]
```diff
@@ -129,19 +144,13 @@ a:focus {
...
-  display: block;
-  margin: auto;
-}
-
 #captchaImg {
   height: 120px;
   width: 220px;
   min-width: 0px;
   max-width: 220px;
+  display: block;
```
Comment (nmattia) [inline src/frontend/src/styles/main.css:152]: The top-level `img` selector was removed, so I moved this here

File: src/frontend/src/flows/manage.ts [hunk 2]
```diff
@@ -93,10 +93,8 @@ const style = () => html`<style>
 // that they add a recovery device. If the user _does_ have at least one
 // recovery device, then we do not display a "nag box", but we list the
 // recovery devices.
-const pageContent = (
-  userNumber: bigint,
-  devices: DeviceData[]
-) => html` ${style()}
+const pageContent = (userNumber: bigint, devices: DeviceData[]) => html`
```
Comment (nmattia) [inline src/frontend/src/flows/manage.ts:96]: Not sure why `npm format` reformatted this

File: src/frontend/src/components/loader.ts [hunk 3]
```diff
@@ -16,6 +16,8 @@ const loader = () => html`<style>
       width: 125px;
       min-width: 125px;
       max-width: calc(100vw - 1rem);
+      margin: auto;
+      display: block;
```
Comment (nmattia) [inline src/frontend/src/components/loader.ts:20]: The top-level `img` selector was removed, so I moved this here

**Comment by nomeata** [comment]
Fancy video!

File: src/frontend/src/components/footer.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { html } from "lit-html";
+
+export const footer = html`<footer>
+  <div class="powered-by">
```
Comment (peterpeterparker) [inline src/frontend/src/components/footer.ts:4]: Can't the `<div class="powered-by">` be spared?

File: src/frontend/src/components/footer.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+    <a class="powered-by-link" href="https://dfinity.org" target="_blank"
```
Comment (peterpeterparker) [inline src/frontend/src/components/footer.ts:5]: should we add an accessibility hint like `aria-label="DFINITY homepage"`?

File: src/frontend/src/components/navbar.ts [hunk 6]
```diff
@@ -3,5 +3,5 @@ import { aboutLink } from "../components/aboutLink";
 import { faqLink } from "../components/faqLink";
 
 export const navbar = html`<div id="navbar">
```
Comment (peterpeterparker) [inline src/frontend/src/components/navbar.ts:5]: if navigation, the element should be a `nav`

File: src/frontend/src/components/navbar.ts [hunk 7]
```diff
@@ -3,5 +3,5 @@ import { aboutLink } from "../components/aboutLink";
...
-  ${aboutLink} &middot; ${faqLink}
+  <div class="nav-links">${aboutLink} &middot; ${faqLink}</div>
```
Comment (peterpeterparker) [inline src/frontend/src/components/navbar.ts:6]: can't `<div class="nav-links">` be spared?

**Review by peterpeterparker — COMMENTED** [review]
Just a couple of things I noticed, all good

**Comment by nmattia** [comment]
> Fancy video!

I know!

File: src/frontend/src/styles/main.css [hunk 8]
```diff
@@ -33,6 +33,21 @@ body {
...
+}
+
+.powered-by-link {
+  padding: 0.5em;
+  display: inline-block;
+}
+
+.powered-by-link:hover,
+.powered-by-link:focus {
+  background: var(--grey-050);
```
Comment (peterpeterparker) [inline src/frontend/src/styles/main.css:47]: you can add a small transition to make it smooth, something like

```
.powered-by-link {
    transition: background 0.2s;
```

**Review by peterpeterparker — APPROVED** [review]
look good thx

File: src/frontend/src/components/navbar.ts (hunk 7 above)
Comment (nmattia) [reply src/frontend/src/components/navbar.ts:6]: It's used to text-align: center

File: src/frontend/src/styles/main.css (hunk 8 above)
Comment (nmattia) [reply src/frontend/src/styles/main.css:47]: Nice! Added.

**Pull request merged**